src/ToggleButton/tests.jsx

```diff
@@ -0,0 +1,202 @@
+/* eslint-env node, mocha */
+/* global expect */
+/* eslint no-console: 0*/
+/* eslint-disable no-magic-numbers, no-multi-str, no-unused-expressions */
+
+
+import React               from 'react';
+import { shallow, mount }  from 'enzyme';
+
+import Icon                from '../Icon';
+
+import ToggleButton        from './index';
+
+
+describe( 'ToggleButton', () =>
+{
+    let instance;
+    let wrapper;
+
+    beforeEach( () =>
+    {
+        wrapper = shallow( <ToggleButton /> );
+        instance = wrapper.instance();
+    } );
+    describe( 'label', () =>
+    {
+        it( 'should be undefined by "default"', () =>
```

Comment:
The quote marks don't make sense to me here (`"`). I think it would read better as simply `should be undefined by default`